server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraCacheConfiguration.java

```diff
@@ -33,25 +33,12 @@
 public class CassandraCacheConfiguration {
 
     public static class Builder {
-        private static final Duration DEFAULT_READ_TIMEOUT = Duration.ofMillis(100);
-        private static final Duration MAX_READ_TIMEOUT = Duration.ofHours(1);
         private static final Duration DEFAULT_TTL = Duration.ofDays(7);
         private static final int DEFAULT_BYTE_THRESHOLD_SIZE = 8 * 1024;
 
-        private Optional<Duration> readTimeout = Optional.empty();
         private Optional<Integer> sizeThresholdInBytes = Optional.empty();
         private Optional<Duration> ttl = Optional.empty();
 
-        public Builder timeOut(Duration timeout) {
```

Comment:
Don't forget to remove `cache.cassandra.timeout`  in documents, if it is not used